Modules/ThirdParty/HDF5/src/itkhdf5/config/cmake/ConfigureChecks.cmake

```diff
@@ -290,7 +290,9 @@ macro (C_RUN FUNCTION_NAME SOURCE_CODE RETURN_VAR)
 
     if (${COMPILE_RESULT_VAR})
       if (${RUN_RESULT_VAR} MATCHES 0)
+        #[[ITK -- start Lookingin into an upstream PR to remove this problematic code
         set (${RUN_RESULT_VAR} 1 CACHE INTERNAL "Have C function ${FUNCTION_NAME}")
```

Comment:
[HDFGroup/hdf5@`0afdf98` (#1332)](https://github.com/HDFGroup/hdf5/pull/1332/commits/0afdf9821c55fc42025db4827ea94dc150c2e2f8) had 2 lines changed. It the second change forgotten here?